src/Akeneo/Pim/Structure/Component/Normalizer/ExternalApi/AttributeNormalizer.php

```diff
@@ -35,6 +40,9 @@ public function normalize($attribute, $format = null, array $context = [])
             $normalizedAttribute['labels'] = (object) $normalizedAttribute['labels'];
         }
 
+        // API-1253: Add read-only attribute group labels inside attribute
```

Comment:
```suggestion
        // Add read-only attribute group labels inside attribute
```
No need to put the issue number (not like what we do with some bugs to retrieve the detailed issue)